examples/linuxbrige_eth1_up_port_vlan.yml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Thank you for an example. Please add this also as a test to https://github.com/nmstate/nmstate/blob/master/tests/integration/examples_test.py to ensure the example works.

Reply:
Done.